src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
```

Comment:
Ah, it's fine then. I'd only use explicit arguments instead of `arguments[1/2]`: 

```js
subscribe(observer, onError, onComplete) { ...
```